templates/settings.yaml.erb

```diff
@@ -74,6 +74,7 @@
 
 :dynflow:
```

Comment:
Just a general question, also to @adamruzicka. Why do we configure the dynflow pool size and redis URL here when we also have a worker config? Does the main Foreman ever use these settings?

Reply:
Actually it doesn't need it at all, although do we want to configure the URL in the job config itself?

Reply:
For the record it would be harder to put it in sidekiq process setting as it is sidekiq config file now. So to put other optios (like sidekiq url) we would need to parse it by ourselves.

Reply:
just wondering, would that work with https://github.com/theforeman/foreman/pull/7153#discussion_r356590683 ?